mytk/controls.py

```diff
@@ -7,21 +7,42 @@ def __init__(
         self, maximum=100, width=200, height=20, orient=HORIZONTAL, delegate=None
     ):
         super().__init__()
+        self.minimum = 0
         self.maximum = maximum
-        self.width = width
-        self.height = height
+        self._width = width
+        self._height = height
```

Comment:
this could lead to some unexpected behaviors I think. If you put some width and height and want to change them before creating the widget, you would most likely go through the getters/setters and since they are programmed to ignore if the widget isn't created then nothing would happen, so I'm gonna put a code suggestion on the getters/setters to try and mitigate this, but I don't know if it's the cleanest code.

Reply:
I removed the setters and getters.  This whole "pre-widget creation" properties aspect needs some thinking through.

Reply:
See new modifications: I am now using self._widget_args and passing them to the widget (not fully implemented in all, but at least in Label and Box).